simpledotorg/simple-server
# Remove tailwind css and sakura css

**Story card:** No card

## Because

These CSS files appear to be old dead-ends to experimentation. They're crufty and worth removing.

## This addresses

Removing unused code. Cleaning.

## Test instructions

I've navigated around the DEV environment and I'm 99.9% sure these classes aren't used. If you navigate the dashboard, progress tab, or other webviews, we shouldn't see any chages.
